charts/scalardb-graphql/values.yaml

```diff
@@ -0,0 +1,150 @@
+  logLevel: INFO
+  # -- Path component of the URL of the GraphQL endpoint. The default is /graphql.
+  path: /graphql
+  # -- Comma-separated list of namespaces of tables for which the GraphQL server generates a schema.
+  namespaces: ""
+  # -- Whether the GraphQL server serves GraphiQL IDE. The default is true.
+  graphiql: "true"
+
+image:
+  # -- Docker image reposiory of Scalar DB GraphQL.
+  repository: ghcr.io/scalar-labs/scalardb-graphql
+  # -- Specify a image pulling policy.
+  pullPolicy: IfNotPresent
+  # -- Docker tag of the image.
+  tag: 3.5.0
+
+# -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
+imagePullSecrets: [name: reg-docker-secrets]
+
+strategy:
+  rollingUpdate:
+    # -- The number of pods that can be created above the desired amount of pods during an update
+    maxSurge: 25%
+    # -- The number of pods that can be unavailable during the update process
+    maxUnavailable: 25%
+  # -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
+  type: RollingUpdate
+
+service:
+  # -- service types in kubernetes.
+  type: ClusterIP
+  # -- Scalar DB GraphQL server port.
+  port: 8080
+
+ingress:
+  # -- Enable ingress resource.
+  enabled: true
+  # -- The ingress class name. Specify "alb" for AWS Application Load Balancer.
+  className: ""  # nginx / alb / ...
+  # -- The class-specific annotations for the ingress resource.
+  annotations:
+    # AWS
+    alb.ingress.kubernetes.io/scheme: internal
+    alb.ingress.kubernetes.io/target-group-attributes: stickiness.enabled=true,stickiness.lb_cookie.duration_seconds=60
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/healthcheck-path: /graphql?query=%7B__typename%7D
+
+    # Azure
+    # kubernetes.io/ingress.class: azure/application-gateway
+    # appgw.ingress.kubernetes.io/cookie-based-affinity: "true"
+
+    # nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/session-cookie-path: /
+    nginx.ingress.kubernetes.io/affinity: cookie
+    nginx.ingress.kubernetes.io/session-cookie-name: INGRESSCOOKIE
+    nginx.ingress.kubernetes.io/session-cookie-hash: sha1
+    nginx.ingress.kubernetes.io/session-cookie-max-age: "300"
+
+  # -- List of rules that are handled with the the ingress.
+  hosts:
+    - host: ""
+      paths:
+        - path: /graphql
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+serviceMonitor:
+  # -- Enable metrics collect with prometheus.
+  enabled: false
+  # -- Custom interval to retrieve the metrics.
+  interval: "15s"
+  # -- Which namespace prometheus is located. by default monitoring.
+  namespace: monitoring
+
+prometheusRule:
+  # -- Enable rules for prometheus.
+  enabled: false
+  # -- Which namespace prometheus is located. by default monitoring.
+  namespace: monitoring
```

Comment:
It seems that there is no `prometheusrules.yaml` file (The manifest of `PrometheusRule` resource).
Is there any plan to add `prometheusrules.yaml` to alert about Scalar DB GraphQL Server using Prometheus Operator in the future?

If there is no plan, I think this configuration is not necessary.